compliance_checker/cf/cf_1_7.py

```diff
@@ -749,23 +844,39 @@ def check_grid_mapping(self, ds):
                                 str(crs_error)
                             )
                         )
-                    else:
-                        test_ctx.score += 1
-                    test_ctx.out_of += 1
-
-            # existence_conditions
+                        test_ctx.out_of += 1                                   
+            test_ctx.score += 1
+ 
+            # existence_conditions           
             exist_cond_1 = (
                 self._check_gmattr_existence_condition_geoid_name_geoptl_datum_name(var)
             )
-            test_ctx.assert_true(exist_cond_1[0], exist_cond_1[1])
+            if exist_cond_1[0] == False:
+                test_ctx.messages.append("Both geoid_name and geopotential_datum_name "
```

Comment:
I think we already have this check defined here: https://github.com/ioos/compliance-checker/blob/master/compliance_checker/cf/cf_1_7.py#L568-L586

Reply:
removed the IF conditional statement and used the tests' results from  _check_gmattr_existence_condition_ell_pmerid_hdatum and _check_gmattr_existence_condition_geoid_name_geoptl_datum_name